src/templates/components/AdvertisementLabel.svelte

```diff
@@ -0,0 +1,28 @@
+<div class="container">
+	<div class="text">Advertisement</div>
+</div>
+
+<style lang="scss">
+	@import './fonts/Sans.css';
+
+	.container {
```

Comment:
I think the label won't be aligned with the ad on larger screens without the helpers media queries

Reply:
You're right the label is not aligned. I tried it with the media queries but it didn't work. I had to add the padding to replicate the behaviour of the live site.

On a separate note, I notice the layers don't behave correctly with the media queries. I removed them and now I believe the behaviour is aligned to legacy

Reply:
Without the media queries the ad won't be constrained to certain widths at certain breakpoints like the current fabric is?

Reply:
It's a bit confusing because fixed ads behave differently to parallax ads. The background image of the parallax ads is controlled by the commercial code and is outside this template so doesn't respect these media queries.

I've reinstated the media queries, but for parallax ads I've allowed the ad label to go full-width and used padding to place it in the correct position on large screens